<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeWindowClass</class>
 <widget class="QMainWindow" name="HomeWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HomeWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="loginBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtFirstName">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="userPassword">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>First Name </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ID Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtLastName">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>388</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuF_ile">
    <property name="title">
     <string>F&amp;ile</string>
    </property>
   </widget>
   <addaction name="menuF_ile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="homewindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>loginBtn</sender>
   <signal>clicked()</signal>
   <receiver>HomeWindowClass</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
